packages/node-core/src/indexer/mmr.service.ts

```diff
@@ -28,6 +30,7 @@ export class MmrService implements OnApplicationShutdown {
   private isShutdown = false;
   private isSyncing = false;
   private _mmrDb?: MMR;
+  private _mmrPlainDb?: MMR;
```

Comment:
From what I can tell, this is only needed for the mmrQuery controller. I think a simpler option would be to have a new service that is read only and doesn't use a cache version.